hub/src/main/java/ch/prevo/open/hub/match/Match.java

```diff
@@ -0,0 +1,21 @@
+package ch.prevo.open.hub.match;
+
+import ch.prevo.open.encrypted.model.InsurantInformation;
+
+public class Match {
+    private final InsurantInformation exit;
+    private final InsurantInformation entry;
+
+    public Match(InsurantInformation exit, InsurantInformation entry) {
```

Comment:
I was wondering if we should prevent the possibility of creating a match with theoretically an exit and an entry of different oasi numbers? (from what I've read regarding throwing exceptions in a constructor, it would require the class to be final, for security reason)

Reply:
Changed Match to contain only oasi + the two uids